types/react-native-svg-animated-linear-gradient/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for react-native-svg-animated-linear-gradient 0.4
+// Project: https://github.com/FullstackStation/react-native-svg-animated-linear-gradient
+// Definitions by: Federico Grandi <https://github.com/EndBug>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export interface GradientProps {
+    primaryColor?: string;
+    secondaryColor?: string;
+    duration?: number;
+    width?: number | string;
+    height?: number | string;
+    x1?: string;
+    y1?: string;
+    x2?: string;
+    y2?: string;
+    useNativeDriver?: boolean;
```

Comment:
```suggestion
    primaryColor?: string | undefined;
    secondaryColor?: string | undefined;
    duration?: number | undefined;
    width?: number | string | undefined;
    height?: number | string | undefined;
    x1?: string | undefined;
    y1?: string | undefined;
    x2?: string | undefined;
    y2?: string | undefined;
    useNativeDriver?: boolean | undefined;
```
[you can optionally consider to support usage within projects with option exact optional property types enabled](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjYtJ2n76r3AhVMpYsKHebVAE4QFnoECAcQAQ&url=https%3A%2F%2Fwww.typescriptlang.org%2Fplay%2F4-4%2Fnew-ts-features%2Fexact-optional-properties.ts.html&usg=AOvVaw1ruqglvzjmZ9g_CPsbZwbr)

Reply:
Seems like a good idea!